dashboard-for-desktop/articles/dashboard-designer/provide-data/binding-to-federated-data-source.md

```diff
@@ -5,72 +5,77 @@ author: Margarita Zakhodyaeva
 
 # Binding to Federated Data Source
 
-A _federated data source_ combines multiple data sources (except OLAP) in one.
+You can create a federated data source if a dashboard contains at least one data source. Create a federated query where you specify how to federate existing data sources.
 
-## Overview
+You can create federated queries of the following types:
 
-You can create a federated data source if a dashboard contains at least one data source. To do this, create a query where you specify how to federate existing data sources. The federated query can be one of the following types:
+- **Join**
 
-### Join
-Combines rows from two or more tables based on a column they share. The join type specifies records that have matching values in both tables.
+  Combines rows from two or more tables based on a clause. 
 
-### Union
-Combines rows from two or more tables into one data set and removes duplicate rows in merged tables. You can only create a union query for data sources that contain columns with the same name. Data types of such columns should be [implicitly converted](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions#implicit-conversions).
+- **Union and Union All**
 
-### Union All
-Operates like **Union**, but duplicates rows from different tables when they contain the same data.
-
-### Transformation
-Unfolds complex objects and flattens them in a simple data set. The dashboard does not support data source fields that store collections, so you need to transform the data source into a new flattened data table.
+  The **Union** query combines rows from two or more tables into one data set and removes duplicate rows in merged tables. The **UnionAll** query does the same, except it doesn't remove duplicated rows. You can create a union query for data sources if data types of their columns are [implicitly converted](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions#implicit-conversions). 
+- **Transformation**
 
+  If a data source contains a complex column (an object), you can transform its properties to display them as separate columns in a flattened view. 
```

Comment:
Does "them" refer to all columns, or the complex column? For example, do you mean:

`If a data source contains a complex column (an object), you can transform its properties to display it as a separate column in a flattened view.`

Or if "them" refers to all columns:

`If a data source contains a complex column (an object), you can transform its properties to display columns separately in a flattened view.`

Reply:
```suggestion
  If a data source contains a complex column (an object), you can transform its properties to display columns separately in a flattened view.
```